students.txt

```diff
@@ -12,7 +12,9 @@ List of students
 5. Dima P.
 6. Vladimir S.
 7. Daniil I.
-8. Daria P. 
+8. Daria Perepichko
+I like drawing, dancing, traveling, reading and active sports.
```

Comment:
Good, but can you use a list to name all your hobbies?
Use this as a template:
- my hobby 1
- my hobby 2
- ...